extensions/cornerstone/src/hps/mammo.ts

```diff
@@ -0,0 +1,57 @@
+export const mgCC = {
+  id: 'mgCC',
+  name: 'mammo-cc',
+  imageLoadStrategy: 'interleaveTopToBottom',
+  protocolMatchingRules: [
+    { attribute: 'ModalitiesInStudy', constraint: { contains: ['MG', 'DX'] } },
```

Comment:
Hmm, this probably needs a better rule so it doesn't trigger too often.  We really need to add a way to have a constraint X and constraint Y   or X2 and Y2.  DX is probably too aggressive by itself